service.autostop/addon.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon  id="service.autostop" name="Autostop" version="1.0.0" provider-name="jeff@thebinks.com">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+  </requires>    
+  
+  <extension point="xbmc.service" library="service.py"/>
+
+  <extension point="xbmc.python.library" library="utilities.py"/>
```

Comment:
Why do you expose this one as a library? If you only use it internally, there is no need to do this.

Reply:
This is called externally  typically via favourites.xml or keyboard.xml.  It is like a sleep button on an alarm clock.